contracts/utils/LineLib.sol

```diff
@@ -84,4 +85,20 @@ library LineLib {
             address(this).balance;
     }
 
+
+    /**
+     * @notice - Helper function to safely transfer Eth using native call
+     * @param recipient - address of the recipient
+     * @param value - value to be sent (in wei)
+     * @param errorMessage - the reason for the revert
+    */
+    function _safeTransferFunds(
+        address recipient,
+        uint256 value,
+        string memory errorMessage
```

Comment:
id prefer if we returned true/false and handled errors elsewhere. Or figure out how to pass custom errors instead of strings (wouldnt be able to pass custom data into custom error though)

Reply:
yup, makes sense